extensions/wikia/ARecoveryEngine/templates/ARecoveryEngineApiController_getBootstrap.mustache

```diff
@@ -1,43 +1,72 @@
 <!-- Begin recovery -->
 <script type="text/javascript">
-require([
-    "ext.wikia.adEngine.adContext"
-], function (adContext) {
-
-    var context = adContext.getContext();
-    if (context.opts.sourcePointRecovery === true) {
-        {{{code}}}
-
-        window._sp_ = window._sp_ || {};
-        window._sp_.config = window._sp_.config || {};
-        window._sp_.config.account_id = 106;
-        window._sp_.config.publisher_base = '{{domain}}/{{cs_endpoint}}';
-        window._sp_.config.enable_rid = true;
-        window._sp_.config.content_control_callback = function() {
-            var logGroup = 'ext.wikia.aRecoveryEngine.recovery.bootstrap';
-            if (!window.wgUserName) {
-                document.getElementById('WikiaArticle').style.display = 'none';
-                document.getElementById('WikiaArticleMsg').style.display = 'block';
-                window.addEventListener('load', function(){
-                    require(['ext.wikia.aRecoveryEngine.recovery.helper'], function (helper) {
-                        'use strict';
-                        helper.track('callback');
-                    });
-                });
-            } else {
-                window.Wikia.log(['isContentControlEnabled', false], 'debug', logGroup);
-            }
-        };
-
-        window._sp_.bootstrap('{{domain}}/api/v1/ARecoveryEngine/delivery');
-
-        window.addEventListener('load', function(){
-            require(['ext.wikia.aRecoveryEngine.recovery.helper'], function (helper) {
-                'use strict';
-                helper.verifyContent();
-            });
-        });
-    }
-});
+	require([
+		"ext.wikia.adEngine.adContext"
+	], function (adContext) {
+		var context = adContext.getContext(),
+		logGroup = 'ext.wikia.aRecoveryEngine.recovery.bootstrap';
+
+		if (context.opts.sourcePointBootstrap === true) {
+			window._sp_ = window._sp_ || {};
+			window._sp_.config = window._sp_.config || {};
+			window._sp_.config.account_id = 106;
+
+			{{{code}}}
+
+			window._sp_.config.content_control_callback = function () {
+				window.Wikia.log('content_control_callback called', 'debug', logGroup);
```

Comment:
A comment from yourself: use `wikia.log` "ENUMs" instead of hard coded strings of log levels ;)